frontend/smart-kickers-game/src/components/Heatmap/Heatmap.css

```diff
@@ -0,0 +1,22 @@
+.heatmap-container {
+  background-image: url('../../assets/images/pitch.png');
+  background-size: 100% 100%;
+  width: 700px;
+  height: auto;
+  transform: scaleX(-1);
+}
+
+.heatmap-container > div > div > div > div {
```

Comment:
Maybe you can avoid that long nesting with class and some JS.

If not, let's say an external library required it.

Reply:
A comment describing that nesting is necesarry was added. Tried changing it with css without satisfying result.